src/client/Client.h

```diff
@@ -1563,7 +1565,7 @@ class Client : public Dispatcher, public md_config_obs_t {
 
   void fill_dirent(struct dirent *de, const char *name, int type, uint64_t ino, loff_t next_off);
 
-  int _opendir(Inode *in, dir_result_t **dirpp, const UserPerm& perms);
+  int _opendir(Inode *in, dir_result_t **dirpp, const UserPerm& perms, int fd);
```

Comment:
```suggestion
  int _opendir(Inode *in, dir_result_t **dirpp, const UserPerm& perms, int fd = -1);
```

so that there is no need to modify existing usages

Reply:
Done